cmd/semdiffstat/main.go

```diff
@@ -49,22 +51,23 @@ func main() {
 		log.Fatalf("could not parse Go files %v and %v: %v\n", aName, bName, err)
 	}
 
+	aline := alignLines(changes)
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 0, ' ', 0)
 	for _, c := range changes {
-		// TODO: align | characters
 		// TODO: general UI improvements
-		if c.Inserted {
-			fmt.Printf("%v (inserted) %s\n", c.Name, pluses(c.InsLines))
-			continue
-		}
-		if c.Deleted {
-			fmt.Printf("%v (deleted) %s\n", c.Name, minuses(c.DelLines))
-			continue
-		}
+		//if c.Inserted {
```

Comment:
Let's keep the annotations, but put them at the end. Something like:

```go
var annotation string
switch {
  case c.Inserted:
    annotation = " (inserted)"
  case c.Deleted:
    annotation = " (deleted)"
}
```

and then in the Fprintf, add a "%s" to the format string like `"%v\t | %*d\t %s%s%s\t\n",` and pass in annotation as an arg after minuses.